Alertinator.php

```diff
@@ -106,6 +190,16 @@ protected function alert($exception, $alertee) {
          }
       }
    }
+   
+   /**
+    * We sometimes need to modify the Exception's message for threshold alerts.
+   */
+   private function prependExceptionMessage($e, $newMessage) {
+      $oldMsg = $e->getMessage();
+      $newMsg = $newMessage . $oldMsg;
```

Comment:
If we're expecting $newMessage . $e->getMessage(), I think we should change the argument order to reflect that.